qiskit/algorithms/gradients/base_estimator_gradient.py

```diff
@@ -103,7 +103,7 @@ def _run(
         self,
         circuits: Sequence[QuantumCircuit],
         observables: Sequence[BaseOperator | PauliSumOp],
-        parameter_values: Sequence[Sequence[float]],
+        parameter_values: Sequence[Sequence[complex]],
```

Comment:
Why are these complex and not `float` anymore? 🤔

Reply:
Actually, I'm not sure. @dlasecki suggested to change it. It's maybe used in VarQTE?
https://github.com/Qiskit/qiskit-terra/pull/8688#discussion_r970458315

Reply:
I thought that in general these might be complex? But now also not sure if the execution path would handle that.

Reply:
I talked to @dlasecki . He said VarQTE doesn't use complex parameter values. Let's make them back to `float`